app/views/layouts/application.html.erb

```diff
@@ -5,63 +5,55 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
-<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
   </head>
-<body>
+  <body>
     <header class="p-3 bg-warning text-white">
       <div class="container">
         <nav class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
-          
-   
-          
+          <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0 display-8">
+            <li><%= link_to 'MyOrders', orders_path,
+           class: 'nav-link px-2 text-white justify-content-center' %></li>
+          </ul>
           <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0 display-6">
-           <li><%= link_to 'GLOVO', root_path, 
+            <li><%= link_to 'GLOVO', root_path, 
            class: 'nav-link px-2 text-info justify-content-center' %></li>
           </ul>
-
           <ul class="nav col-12 col-lg-auto mb-2  mb-md-0">
-          <% if user_signed_in? %>
-            
-           
-            
-            <div style="width: 33%" class="hehe dropdown d-flex justify-content-end">
-        <a class="hehe-link tn dropdown-toggle text-white" role="button" id="dropdownMenuLink" data-bs-toggle="dropdown" aria-expanded="false">
-          <div class='h5 d-inline text-success '><%= current_user.first_name %></div>
-        </a>
-        <ul class="dropdown-menu" aria-labelledby="dropdownMenuLink">
-          <li><%= button_to 'Logout', destroy_user_session_path, method: :delete, class: "dropdown-item" %></li>
-          <li><%= link_to 'Edit profile', edit_user_registration_path(current_user), class: "dropdown-item" %></li>
-        </ul>
-      </div>
+            <% if user_signed_in? %>
```

Comment:
can you move it to a separate file _navbar? Try to make this file as small as possible